project_transportonly/dimensions/enduses.csv

```diff
@@ -0,0 +1,13 @@
+id,name,fuel_id,unit
+electricity_ev_ldv_home_l1,ev_ldv_home_l1,electricity,MWh
+electricity_ev_ldv_home_l2,ev_ldv_home_l2,electricity,MWh
+electricity_ev_ldv_work_l1,ev_ldv_work_l1,electricity,MWh
+electricity_ev_ldv_work_l2,ev_ldv_work_l2,electricity,MWh
+electricity_ev_ldv_public_l2,ev_ldv_public_l2,electricity,MWh
+electricity_ev_ldv_public_dcfc,ev_ldv_public_dcfc,electricity,MWh
+electricity_ev_ldv_enroute_dcfc,ev_ldv_enroute_dcfc,electricity,MWh
+electricity_ev_mhdv_depot_ac,ev_mhdv_depot_ac,electricity,MWh
+electricity_ev_mhdv_depot_dc,ev_mhdv_depot_dc,electricity,MWh
+electricity_ev_mhdv_opportunity_dc,ev_mhdv_opportunity_dc,electricity,MWh
+electricity_ev_mhdv_enroute_dc,ev_mhdv_enroute_dc,electricity,MWh
+electricity_rail_transit,Electric Transit Rail,electricity,MWh
```

Comment:
Sorry I didn't catch this in the main project, but should EVs have pretty names like "Electric Transit Rail" does, e.g., "EV M/HDV Enroute DC"?